provider/bikeshed/Cargo.toml

```diff
@@ -0,0 +1,100 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_datagen_bikeshed"
```

Comment:
We were trying to decide between `icu_provider_source` and `icu_provider_transform`?

Reply:
`icu_provider_datagen` was also in the mix I think